src/home_robot_hw/home_robot_hw/env/stretch_abstract_env.py

```diff
@@ -117,6 +117,10 @@ def __init__(
         Based on this code:
         https://github.com/hello-robot/stretch_ros/blob/master/hello_helpers/src/hello_helpers/hello_misc.py
         """
+        # Deprecation warning
+        rospy.logwarn(
+            "'StretchEnv' is being deprecated. Use 'home_robot_hw.StretchClient' to interact with the robot."
```

Comment:
wait this whole env is being deprecated?

Reply:
That was my understanding, as all of the functionality this layer of abstraction provides is now inside `StretchClient`.
My original thought was that the hardware envs would directly inherit from `home_robot.Env`, and include `StretchClient` as a member variable (submodule), but open to discussion.
Maybe when I'm adapting the previous hardware envs to use the new interface, I can see if duplicated code end up existing across the hardware envs, then recreate `StretchEnv` to contain these duplications.